settings/blt.settings.php

```diff
@@ -229,7 +249,11 @@
  * Keep this code block at the end of this file to take full effect.
  */
 if ($is_local_env) {
-  // Load local machine settings.
+  // Load local settings for all sites.
+  if (file_exists(DRUPAL_ROOT . "/sites/settings/local.settings.php")) {
+    require DRUPAL_ROOT . "/sites/settings/local.settings.php";
```

Comment:
Should probably make sure this will not be added to deploy artifacts. Looks like these should handle it - https://github.com/acquia/blt/tree/8.x/scripts/blt/deploy